src/objectivec/include/ort_genai_objc.h

```diff
@@ -309,13 +320,22 @@ typedef NS_ENUM(NSInteger, OGAElementType) {
 - (nullable OGATensor*)getOutput:(NSString*)name error:(NSError**)error;
 
 /**
- * Retrieve the sequence at the given index.
+ * Retrieve the sequence data at the given index.
+ * @param index The index needed.
+ * @param error Optional error information set if an error occurs.
+ * @return The sequence data at the given index, or nil if an error occurs.
+ */
+- (nullable const int32_t*)sequenceDataAtIndex:(size_t)index
+                                         error:(NSError**)error;
+
+/**
+ * Retrieve the sequence count at the given index.
  * @param index The index needed.
  * @param error Optional error information set if an error occurs.
- * @return The sequence at the given index, or nil if an error occurs.
+ * @return The sequence length at the given index, or nil if an error occurs.
```

Comment:
```suggestion
 * @return The sequence count at the given index, or size_t(-1) if an error occurs.
```

"count" for consistency with `OGASequences sequenceCountAtIndex`